src/messages/layout.js

```diff
@@ -8,3 +8,36 @@ export const DUPLICATE_XPI_ENTRY = {
     with the same name. This practice has been banned. Try unzipping
     and re-zipping your add-on package and try again.`),
 };
+
+export const TYPE_NO_INSTALL_RDF = {
+  code: 'TYPE_NO_INSTALL_RDF',
+  message: _('install.rdf was not found'),
+  description: _(singleLineString`The type should be determined by
```

Comment:
Sorry, I can't quite make sense of this description. It's saying "No install.rdf was found so we're guessing your add-on type, but we can't be sure.", right?